libs/src/LSPlugin.ts

```diff
@@ -183,6 +183,16 @@ export interface IAppProxy {
   showMsg: (content: string, status?: 'success' | 'warning' | string) => void
   setZoomFactor: (factor: number) => void
 
+  registerUIItem: (
+    type: 'toolbar' | 'pagebar',
+    opts: { key: string, template: string }
+  ) => boolean
+
+  registerPagebarMenuItem: (
```

Comment:
It will be added to the page's menu, right? How about renaming it to registerPageMenuItem?

Reply:
COOL, shortness is better!